collectors/ebpf.plugin/ebpf.c

```diff
@@ -129,7 +129,7 @@ ebpf_module_t ebpf_modules[] = {
       .update_time = 1, .global_charts = 1, .apps_charts = CONFIG_BOOLEAN_NO, .mode = MODE_ENTRY,
       .optional = 0, .apps_routine = ebpf_fd_create_apps_charts, .maps = NULL,
       .pid_map_size = ND_EBPF_DEFAULT_PID_SIZE, .names = NULL, .cfg = &fd_config,
-      .config_file = NETDATA_SYNC_CONFIG_FILE},
+      .config_file = NETDATA_FD_CONFIG_FILE},
```

Comment:
Just an FYI that I noticed while doing hard IRQs that many other entries in this table also use NETDATA_SYNC_CONFIG_FILE incorrectly. I'll fix that in my PR as a separate commit or make a new PR.

Reply:
Than you, I fixed with last commit.